### PR TITLE
*: fix forward report and use more forward connections

### PR DESCRIPTION
**Pull request by BusyJay**
### What problem does this PR solve?

Problem Summary:

Too small value can lead to imbalance load.

The new panel looks like:
image

### Check List 

Tests 

- Unit test
- Integration test
- Manual test

### Release note 
- no release note.

### CODE REVIEW

**Review by MyonKeminta — APPROVED** [review]
LGTM

**Review by youjiali1995 — COMMENTED** [review]
Seems each interface has an individual `Proxy`, so the total connections count is interface count * `forward_max_connections_per_address`?

**Comment by BusyJay** [comment]
No, grpc-rs tries to use only one connection if all connecting args are the same.

File: components/pd_client/src/util.rs [hunk 1]
```diff
@@ -43,6 +43,38 @@ const MAX_RETRY_DURATION: Duration = Duration::from_secs(10);
...
 
+pub struct TargetInfo {
+    target_url: String,
+    via: String,
+}
+
+impl TargetInfo {
+    fn new(target_url: String, via: &str) -> TargetInfo {
+        TargetInfo {
+            target_url,
```
Comment (MyonKeminta) [inline components/pd_client/src/util.rs:54]: Is this trimmed?

Comment (BusyJay) [reply components/pd_client/src/util.rs:54]: It's not. And it's expected not to trim as PD only accepts url currently.

**Review by MyonKeminta — APPROVED** [review]
LGTM

**Review by youjiali1995 — APPROVED** [review]

**Pull request merged**